features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -179,7 +179,7 @@
 
         <!-- This configuration is used to specify whether the Service Provider tenant domain should be used when generating
              access token. Otherwise user domain will be used. Currently this value is only supported by the JWTTokenIssuer.-->
```

Comment:
add a comment saying that this config is is enabled  as true from IS 5.12.o

Reply:
Added comment.